src/backend/db/videos.js

```diff
@@ -5,17 +5,158 @@
 
 export const videos = [
   {
-    _id: "Wo5dMEP_BbI",
-    title: "Awesome Video about Coding",
-    description:
-      "At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium voluptatum deleniti atque corrupti quos dolores et quas molestias excepturi sint occaecati cupiditate non provident, similique sunt in culpa qui officia deserunt mollitia animi, id est laborum et dolorum fuga.",
-    creator: "Soham Shah",
+    _id: "okTZb6GkU0U",
+    thumbnail: "https://i.ytimg.com/vi/okTZb6GkU0U/hq720.jpg?sqp=-oaymwEcCNAFEJQDSFXyq4qpAw4IARUAAIhCGAFwAcABBg==&rs=AOn4CLCh7hsAOQ-ZDrxVoY7ObhdAA969bw",
+    title: "HORIZON ZERO DAWN Walkthrough Gameplay Part 1 - Aloy (PS4 Pro)",
+    creatorAvatar: "https://yt3.ggpht.com/ytc/AKedOLTBSwIyFAMOXI0XdsyxQ8NAEI93NI2-UUTA2qM-oQ=s88-c-k-c0x00ffffff-no-rj",
```

Comment:
While setting up the project, you can directly put backend data from dev. Instead of including it in PR